cmd/apiserver-boot/boot/build/build_resource_config.go

```diff
@@ -272,7 +272,7 @@ type resourceConfigTemplateArgs struct {
 var resourceConfigTemplate = `
 {{ $config := . -}}
 {{ range $api := .Versions -}}
-apiVersion: apiregistration.k8s.io/v1beta1
```

Comment:
thx for fixing this, update related yaml files under `example/` please

Reply:
sure